tools/itch5bookdepth.cpp

```diff
@@ -0,0 +1,146 @@
+#include <jb/itch5/compute_book_depth.hpp>
+#include <jb/itch5/process_iostream.hpp>
+#include <jb/book_depth_statistics.hpp>
+#include <jb/fileio.hpp>
+#include <jb/log.hpp>
+
+#include <iostream>
+#include <stdexcept>
+#include <unordered_map>
+
+#include <chrono>
+
+/* Ticket https://github.com/coryan/jaybeams/issues/20
```

Comment:
Please remove the links to the issue.  If we did this for every change the comments will grow tedious.